source/dotnet/wholesale-api/Infrastructure/ServiceBus/IIntegrationEventTopicServiceBusSender.cs

```diff
@@ -18,5 +18,5 @@ namespace Energinet.DataHub.Wholesale.Infrastructure.ServiceBus;
 
 public interface IIntegrationEventTopicServiceBusSender
 {
-    Task SendMessageAsync(ServiceBusMessage message, CancellationToken cancellationToken);
+    Task SendMessageAsync(ServiceBusMessage serviceBusMessage, CancellationToken cancellationToken);
```

Comment:
nit: I personally prefer this 😄 
```suggestion
    Task SendMessageAsync(ServiceBusMessage message, CancellationToken cancellationToken);
```

Reply:
I will leave this for later.